Makefile

```diff
@@ -29,13 +29,13 @@ registry-login:
 tag:
 	$(eval BUILD_ID=$(shell ${CONTAINER_ENGINE} image inspect --format '{{.ID}}' $(IMAGE_NAME) | cut -d ':' -f 2 | cut -c1-12) )
 	${CONTAINER_ENGINE} tag $(IMAGE_NAME) $(IMAGE_URI):$(GIT_REVISION)-$(BUILD_ID)
```

Comment:
Would you mind if we swapped the GIT_REVISION and BUILD_ID in the long tags too?

Reply:
```suggestion
	${CONTAINER_ENGINE} tag $(IMAGE_NAME) $(IMAGE_URI):$(BUILD_ID)-$(GIT_REVISION)
```